pkg/vfs/vfs_unix.go

```diff
@@ -94,6 +94,11 @@ func (v *VFS) Access(ctx Context, ino Ino, mask int) (err syscall.Errno) {
 	}
 
 	err = v.Meta.Access(ctx, ino, uint8(mmask), nil)
+	if err == 0 {
+		if aerr := v.touchAtime(ctx, ino); aerr != 0 {
```

Comment:
Access is used to check permission, we should not update atime

Reply:
True, ext4 doesn't update atime on access(2) as well. Fixed